search-mapper/src/main/java/org/infinispan/search/mapper/session/impl/InfinispanSearchSessionMappingContext.java

```diff
@@ -0,0 +1,14 @@
+package org.infinispan.search.mapper.session.impl;
+
+import java.util.Collection;
+
+import org.hibernate.search.mapper.pojo.session.spi.PojoSearchSessionMappingContext;
+import org.infinispan.search.mapper.scope.SearchScope;
+import org.infinispan.search.mapper.scope.impl.SearchScopeImpl;
+
+public interface InfinispanSearchSessionMappingContext extends PojoSearchSessionMappingContext {
```

Comment:
Right, the session API is never used from other modules at the moment. I could do this refractory now, but I would prefer to follow up. Because we're going to change the session access in the other follow up.